docs/src/starknet/account.md

```diff
@@ -49,28 +55,30 @@ account: https://starkscan.co/search/0x7a949e83b2...
 
  >💡 **Info**
 > When deploying on a Sepolia test network, you can also fund your account with artificial tokens via the [Starknet Faucet](https://starknet-faucet.vercel.app)
-
+>![image](images/starknet-faucet-sepolia.png)
```

Comment:
```
To deploy an account in the next step, you need to prefund it with either STRK or ETH tokens
```

This could say 

```
To deploy an account in the next step, you need to prefund it with either STRK or an equivalent amount of ETH token
```